### ISSUE TITLE
[Bug]: dd() Collapsing Not Working in Laravel Debug Output

### ISSUE THREAD

**Issue by jfBiswajit** [laravel/framework#47342]
### Laravel Version

10.13.1

### PHP Version

8.2.6

### Database Driver & Version

MySQL 8.0.33 for macos13.3 on arm64 (Homebrew)

### Description

The dd() and dump() functions in Laravel do not properly collapse when clicking on the arrow icon in the debug output. The dumped variables remain expanded regardless of the click action.

### Steps To Reproduce

Create a fresh Laravel project with no custom configuration.
Add the following code snippet to a route or controller:

``` PHP
$fruits = [
    'fruits' => [
        'apples' => [
            'red',
            'green',
        ],
        'bananas' => [
            'yellow',
            'green',
        ],
    ],
    'vegetables' => [
        'carrots' => [
            'orange',
            'purple',
        ],
        'spinach' => [
            'green',
        ],
    ],
];
```

```dd($fruits);```
Run the Laravel project and access the route or controller that executes the above code.
Observe the debug output and try to collapse the dumped variables by clicking on the arrow icon.

**Expected Behavior:**
Clicking on the arrow icon next to the dd() or dump() dump should toggle the collapse state of the dumped variables, allowing for a better overview of the debug output.

**Actual Behavior:**
Clicking on the arrow icon does not toggle the collapse state, and the dumped variables remain expanded regardless of the click action.

**Additional Information:**
I have tested the issue on both Chrome and Firefox, as well as in incognito mode, but the behavior remains the same. The bug occurs in a fresh Laravel project without any custom configuration.

Please let me know if there are any further details or steps I can provide to assist in resolving this issue. Feel free to reach out if you need any additional information.

**Comment by kikemaru** [comment]
I don't have that problem. Try deploying the application in a container.

**Comment by Ximerox** [comment]
I also have this issue. Im on version 10.13.5
dd() opens with alle object already expanded. There is no way to collapse them.

**Comment by imerfanahmed** [comment]
Yes I do have the same issue!

**Comment by samsaffre** [comment]
Hello I confirm

To be sure I have created a new blank laravel project and put this code into routes/web.php
`
Route::get('/', function () {
    $test = collect([]);
    for($i=0 ; $i< 100 ; $i++){
        $test->push([1,2,3,4,5]);
    }
    dd($test);
});
`

And dump(output not collapsed)...

**Comment by geovaniralmeida** [comment]
I also went through the same problem. From what I saw, this happened when it was dumped out of an html document, that is, directly from the controller. I believe that symfony/var-dumper version 6.3 has a problem.
The following update should fix.
[https://github.com/symfony/var-dumper/commit/36579a1133f46b5fc0609326a19a47cd62e86675](https://github.com/symfony/var-dumper/commit/36579a1133f46b5fc0609326a19a47cd62e86675)

**Comment by FelipeVeiga** [comment]
same here

**Comment by geovaniralmeida** [comment]
I solved it by requesting the version(v6.2.3) explicitly in composer.json
"symfony/var-dumper": "v6.2.3"

**Comment by jsonimac** [comment]
> I solved it by requesting the version(v6.2.3) explicitly in composer.json "symfony/var-dumper": "v6.2.3"

can you share on how did you request to version(v6.2.3) ?

**Comment by jsonimac** [comment]
I fix it on my end, by installing `composer require --dev symfony/var-dumper` and reloading the page. 
![image](https://github.com/user-attachments/assets/ba8b6658-fdce-41ca-a0d4-5c4222c39595)
![image](https://github.com/user-attachments/assets/644de2e8-78aa-4e51-bf64-9260ee47a62c)
https://github.com/user-attachments/assets/b6c74642-ee86-469c-bfe6-5d245682c578
